pkgs/top-level/darwin-packages.nix

```
@@ -240,5 +241,5 @@ impure-cmds // appleSourcePackages // chooseLibs // {
       };
 
     in
-      nixos.config.system.build.macos-builder-installer;
+      nixos.config.system.build.macos-builder-installer) { config = { }; };
```

```suggestion
      nixos.config.system.build.macos-builder-installer) { modules = [ ]; };
```

Consistent with `nixpkgsFlake.lib.nixosSystem` and `evalModules`. Use `++` in `imports`.

---

Updated :+1: